integration_tests/snaps/simple-rust-with-revision/snapcraft.yaml

```diff
@@ -0,0 +1,13 @@
+name: test-rust-on-revision
```

Comment:
I can fix them all next week. Feel free to leave it, one more, one less, no big difference :)